teamB/jungJaeUk/app.js

```diff
@@ -49,6 +95,83 @@ app.post('/signup', async (req, res) => {
   );
   res.status(201).json({ "message" : "userCreated" });
 });
+//[POST] post add
+app.post('/posts/add', async (req, res) => {
```

Comment:
- 현재 endpoint에서 /add 는 POST 요청 자체가 등록을 요청한다는 의미도 포함되어 있기에 restful 하지 않다고 볼 수 있습니다!
- 해당 endpoint를 GET의 /posts, POST의 /posts 처럼 엔드포인트는 같으나, 요청 메소드가 다르기 때문에 자명한 endpoint를 만들 수 있을 거 같습니다 ~

```suggestion
app.post('/posts', async (req, res) => {
```

이렇듯 자명한 엔드포인트를 작성하는 것에 대해 REST하게 엔드포인트를 작성한다고 합니다.
해당 내용은 이후에 있을 RESTful API 세션에서 다뤄질 예정이니 참고해주세요 :)

Reply:
수정하겠습니다.